frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -129,6 +150,50 @@ const TestQueue: React.FC<Props> = ({ activeFacilityId }) => {
     );
   }
 
+  const showPatientAddedToQueueAlert = (patient: Patient) => {
+    const { type, title, body } = {
+      ...ALERT_CONTENT[QUEUE_NOTIFICATION_TYPES.ADDED_TO_QUEUE__SUCCESS](
+        patient
+      ),
+    };
+    showAlertNotification(type, title, body);
+  };
+
+  const addPatientToQueue = async (patient: Patient) => {
+    if (appInsights) {
+      appInsights.trackEvent({ name: "Add Patient To Queue" });
+    }
+    try {
```

Comment:
I think we may want to add a `finally` block here to `setStartTestPatientId(null);` because we run into issues if we add a patient via the patients page to the test queue


Uploading error toast bug.mov…

Reply:
Great bug find! I'll push an update though it may have to be added in the `catch` block to only reset when there's an error since that id is still used after the mutation succeeds